LocalMultiplayerAgent/MultiplayerSettingsLinuxContainersOnWindowsSample.json

```diff
@@ -1,36 +1,37 @@
 ﻿{
-    "RunContainer": true,
-    "OutputFolder": "C:\\output\\UnityServerLinux",
-    "NumHeartBeatsForActivateResponse": 10,
-    "NumHeartBeatsForTerminateResponse": 60,
-    "TitleId": "",
-    "BuildId": "00000000-0000-0000-0000-000000000000",
-    "Region": "WestUs",
-    "AgentListeningPort": 56001,
-    "ContainerStartParameters": {
-        "ImageDetails": {
-            "Registry": "mydockerregistry.io",
-            "ImageName": "mygame",
-            "ImageTag": "0.1",
-            "Username": "",
-            "Password": ""
-        }
-    },
-    "PortMappingsList": [
-        [
-            {
-                "NodePort": 56100,
-                "GamePort": {
-                    "Name": "game_port",
-                    "Number": 7777,
-                    "Protocol": "TCP"
-                }
-            }
-        ]
-    ],
-    "SessionConfig": {
-        "SessionId": "ba67d671-512a-4e7d-a38c-2329ce181946",
-        "SessionCookie": null,
-        "InitialPlayers": [ "Player1", "Player2" ]
+  "RunContainer": true,
+  "ForcePullFromAcrOnLinuxContainersOnWindows": true,
```

Comment:
I'd make it 'false', since it's the default till now

Reply:
gotcha!